### PR TITLE
GTX-349 - Adds test for sound toggle in game window

### PR DESCRIPTION
**Pull request by MattClare9743**

### CODE REVIEW

File: skyVegasTests/soundCheckGameClient.spec.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+import { launchGame, validateConsoleMessages, login } from "./utilities";
+import * as fs from "fs";
+import { text } from "stream/consumers";
+
+test.describe.configure({ mode: "serial" });
+const consoleMessages: string[] = [];
+let jsonObject;
+let page: Page;
+const textValues = JSON.parse(fs.readFileSync("games.json", "utf-8"));
+textValues.forEach((textValue) => {
```
Comment (gwenso) [inline skyVegasTests/soundCheckGameClient.spec.ts]: suggestion: these names are very vague, how about calling them "games" and "game" i.e.
games.forEach((game ...

File: skyVegasTests/soundCheckGameClient.spec.ts [hunk 2]
```diff
@@ -0,0 +1,53 @@
+import { test, expect, type Page } from "@playwright/test";
+import { launchGame, validateConsoleMessages, login } from "./utilities";
+import * as fs from "fs";
+import { text } from "stream/consumers";
```
Comment (gwenso) [inline skyVegasTests/soundCheckGameClient.spec.ts]: doesn't appear to be needed

File: skyVegasTests/soundCheckGameClient.spec.ts [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+test.describe.configure({ mode: "serial" });
+const consoleMessages: string[] = [];
+let jsonObject;
+let page: Page;
+const textValues = JSON.parse(fs.readFileSync("games.json", "utf-8"));
+textValues.forEach((textValue) => {
+  test.describe(`Testing with text: ${textValue}`, () => {
+    test.beforeAll(async ({ browser }) => {
+      const data = fs.readFileSync("ExpectedSlotConsoleMessages.json", "utf-8");
+      jsonObject = JSON.parse(data);
```
Comment (gwenso) [inline skyVegasTests/soundCheckGameClient.spec.ts]: likewise with this name - this gives no detail as to what this object is. could be something like "expectedMessages"

File: skyVegasTests/soundCheckGameClient.spec.ts (hunk 1 above)
Comment (MattClare9743) [reply skyVegasTests/soundCheckGameClient.spec.ts]: yep I agree, will change them now

**Review by gwenso — APPROVED** [review]

File: skyVegasTests/soundCheckGameClient.spec.ts [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+      await launchGame(page, game);
+      if (expectedMessages.soundCheckMessageToolBarOn) {
+        await(
+            page
+            .frameLocator('#root iframe').locator('i').nth(2)
+            .click());
+        validateConsoleMessages(
+        page,
+        expectedMessages.soundCheckMessageToolBarOff,
+        consoleMessages
```
Comment (AlexeiBlue) [inline skyVegasTests/soundCheckGameClient.spec.ts:36]: suggestion: fix indentation on these three lines

File: skyVegasTests/soundCheckGameClient.spec.ts [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+    test("Test sound toggle", async ({}) => {
+      page.on("console", (msg) => {
+        consoleMessages.push(msg.text());
+      });
+      await launchGame(page, game);
+      if (expectedMessages.soundCheckMessageToolBarOn) {
+        await(
+            page
+            .frameLocator('#root iframe').locator('i').nth(2)
+            .click());
```
Comment (AlexeiBlue) [inline skyVegasTests/soundCheckGameClient.spec.ts:32]: suggestion: move bracket colon ); to next line and align with the await( keyword

File: skyVegasTests/soundCheckGameClient.spec.ts [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+        page,
+        expectedMessages.soundCheckMessageToolBarOff,
+        consoleMessages
+        );
+      }
+      else if(expectedMessages.soundCheckMessageToolBarOff) {
+        await(
+            page
+              .frameLocator('#root iframe').locator('i').nth(2)
+              .click());
```
Comment (AlexeiBlue) [inline skyVegasTests/soundCheckGameClient.spec.ts:43]: suggestion: move bracket colon ); to next line and align with the await( keyword

File: skyVegasTests/soundCheckGameClient.spec.ts [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+          validateConsoleMessages(
+            page,
+            expectedMessages.soundCheckMessageToolBarOn,
+            consoleMessages
+            );
```
Comment (AlexeiBlue) [inline skyVegasTests/soundCheckGameClient.spec.ts:48]: suggestion: move ); to be inline with validateConsoleMessages(

**Pull request merged**